kolibri/core/exams/test/test_exam_api.py

```diff
@@ -276,6 +276,15 @@ def test_logged_in_user_exam_no_update(self):
         })
         self.assertEqual(response.status_code, 403)
 
+    def test_cannot_create_exam_same_title(self):
+        response = self.client.post(reverse("kolibri:core:exam-list"), {
+            "title": "title",
+            "question_count": 2,
+            "active": True,
+            "collection": self.facility.id,
+        })
+        self.assertEqual(response.status_code, 400)
```

Comment:
Could make a further assertion about the form of the validation error raised? (I think this is important because we are relying on the specific error raised on the frontend).